doc/_utils/redirects.py

```diff
@@ -4,455 +4,474 @@
+    ("app_dev/testing_unity_cmock/index", "test_and_optimize/testing_unity_cmock"),
+    ("test_and_optimize/testing_unity_cmock", "test_and_optimize/test_framework/testing_unity_cmock"),
+    ("app_opt", "test_and_optimize/optimizing/index"), # Optimizing application (landing)
+    ("app_dev/optimizing/index", "test_and_optimize/optimizing/index"),
+    ("app_memory", "test_and_optimize/optimizing/memory"), # Memory footpring optimization
+    ("app_dev/optimizing/memory", "test_and_optimize/optimizing/memory"),
+    ("app_power_opt", "test_and_optimize/optimizing/power"), # Power optimization
+    ("app_dev/optimizing/power", "test_and_optimize/optimizing/power"),
+    ("security_chapter","security/security"), # Security (landing)
+    ("security","security/security"),
+    ("ug_tfm", "security/tfm"), # Running applications with Trusted Firmware-M
+    ("app_dev/tfm/index", "security/tfm"),
+    ("app_dev/ap_protect/index", "security/ap_protect"), # Enabling access port protection mechanism
+    ("ug_ble_controller", "protocols/ble/index"), # Bluetooth LE Controller
+    ("protocols/ble/index", "protocols/bt/ble/index"),
+    ("ug_bt_mesh", "protocols/bt_mesh/index"), # Bluetooth Mesh (landing)
+    ("protocols/bt_mesh/index", "protocols/bt/bt_mesh/index"),
+    ("ug_bt_mesh_supported_features", "protocols/bt_mesh/supported_features"), # Supported Bluetooth Mesh features
+    ("protocols/bt_mesh/supported_features", "protocols/bt/bt_mesh/supported_features"),
+    ("ug_bt_mesh_concepts", "protocols/bt_mesh/concepts"), # Bluetooth Mesh concepts
+    ("protocols/bt_mesh/concepts", "protocols/bt/bt_mesh/concepts"),
+    ("ug_bt_mesh_architecture", "protocols/bt_mesh/architecture"), # Bluetooth Mesh stack architecture
+    ("protocols/bt_mesh/architecture", "protocols/bt/bt_mesh/architecture"),
+    ("ug_bt_mesh_configuring", "protocols/bt_mesh/configuring"), # Configuring Bluetooth Mesh in nRF Connect SDK
+    ("protocols/bt_mesh/configuring", "protocols/bt/bt_mesh/configuring"),
+    ("ug_bt_mesh_model_config_app", "protocols/bt_mesh/model_config_app"), # Configuring mesh models using the nRF Mesh mobile app
+    ("protocols/bt_mesh/model_config_app", "protocols/bt/bt_mesh/model_config_app"),
+    ("ug_bt_mesh_fota", "protocols/bt_mesh/fota"), # Performing Device Firmware Updates (DFU) in Bluetooth Mesh
+    ("protocols/bt_mesh/fota", "protocols/bt/bt_mesh/fota"),
+    ("ug_bt_mesh_node_removal", "protocols/bt_mesh/node_removal"), # Removing a node from a mesh network
+    ("protocols/bt_mesh/node_removal", "protocols/bt/bt_mesh/node_removal"),
+    ("ug_bt_mesh_vendor_model", "protocols/bt_mesh/vendor_model/index"), # Creating a new model (landing)
+    ("protocols/bt_mesh/vendor_model/index", "protocols/bt/bt_mesh/vendor_model/index"),
+    ("ug_bt_mesh_vendor_model_dev_overview", "protocols/bt_mesh/vendor_model/dev_overview"), # Vendor model development overview
```

Comment:
```suggestion
    ("ug_bt_mesh_vendor_model_dev_overview", "protocols/bt/bt_mesh/vendor_model/dev_overview"), # Vendor model development overview
```

Reply:
This is incorrect. The edit is needed in the next line, and I have now applied it.

Reply:
I see. Yes, thought it was this line as the redirection failed.